src/bin/facilitator.ts

```diff
@@ -20,6 +20,6 @@
 import facilitator from 'commander';
 
 facilitator
-  .command('init <mosaic-config> <aux-chain-id> <origin-password> <auxiliary-password> <origin-rpc> <auxiliary-rpc> <origin-graph-ws> <origin-graph-rpc> <auxiliary-graph-ws> <auxiliary-graph-rpc> <db-path>', 'initializes the facilitator config')
+  .command('init <token-config> <mosaic-config> <aux-chain-id> <origin-password> <auxiliary-password> <origin-rpc> <auxiliary-rpc> <origin-graph-ws> <origin-graph-rpc> <auxiliary-graph-ws> <auxiliary-graph-rpc> <db-path>', 'initializes the facilitator config')
```

Comment:
what is token config?

Reply:
Her does mosaic-config option needed if gateway-config is present?